src/operators/expression.h

```diff
@@ -121,7 +104,7 @@ class Expression {
   arrow::Datum Evaluate(hustle::Task* ctx, int chunk_id);
 
   // number of chunks present in the input column on which it operates.
-  inline int32_t num_chunks() { return exp_num_chunks_; }
+  [[nodiscard]] inline int32_t num_chunks() const { return exp_num_chunks_; }
```

Comment:
I am wondering, why we may need to use [[nodiscard]] here

Reply:
It's a good-to-have attribute and CLion suggested me to have (with a ClangTidy warning). I just removed the nodiscard, but remain the const